### DISCUSSION TITLE
Help with maui

### DISCUSSION THREAD

**Discussion by OP-EX** [dotnet-bluetooth-le/dotnet-bluetooth-le#741] [General]
Can I have a step by step on how to use this with maui

**Comment by janusw** [comment]
The [usage](https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le#usage) section of the README gives some starting points, we have a sample app (which is Xamarin-only right now), and there are also MAUI apps that use the library, like https://github.com/TimoKinnunen/MauiXAMLBluetoothLE.

The usage of the library with MAUI is basically the same as with Xamarin.

**Comment by janusw** [comment]
Note that by now we also have a basic MAUI sample client (added via #736), see https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/tree/master/Source/BLE.Client/BLE.Client.Maui. HTH.
